### ISSUE TITLE
Option to Use Intel MKL Instead of OpenBLAS

### ISSUE THREAD

**Issue by RoyiAvital** [stepelu/lua-sci#9]
Any chance to have support for Intel MKL instead of OpenBLAS?

Intel MKL is faster and includes many useable algorithms (Accelerated LAPCAK in addition to BLAS).

**Comment by stepelu** [comment]
Yes, you have to make sure the required Blas Intel MKL dynamic libraries are loaded via `ffi.load(...)` from https://luajit.org/ext_ffi_api.html.
See for instance `__init.lua` from https://github.com/uluadist-pkg/uluadist-pkg.github.io/blob/master/pkg/clib_libopenblas/0.2.15 (a zip file), but that relies on how things are handled in ULua so it's probably of limited help.
